graph/typeDefs.graphql

```diff
@@ -1278,17 +1390,38 @@ type RootMutation {
   # Edit a comment
   editComment(id: ID!, asset_id: ID!, edit: EditCommentInput): EditCommentResponse!
 
-  # Sets User status. Requires the `ADMIN` role.
+  # Sets the suspension status on a given user. Requires the `MODERATOR` role.
+  # Mutation is restricted.
+  suspendUser(input: SuspendUserInput!): SuspendUserResponse
+
+  # Sets the suspension status on a given user. Requires the `MODERATOR` role.
   # Mutation is restricted.
-  setUserStatus(id: ID!, status: USER_STATUS!): SetUserStatusResponse
+  unSuspendUser(input: UnSuspendUserInput!): UnSuspendUserResponse
 
-  # Suspends a user. Requires the `ADMIN` role.
+  # Sets the ban status on a given user. Requires the `MODERATOR` role.
   # Mutation is restricted.
-  suspendUser(input: SuspendUserInput!): SuspendUserResponse
+  banUser(input: BanUserInput!): BanUsersResponse
 
-  # Reject a username. Requires the `ADMIN` role.
+  # Sets the ban status on a given user. Requires the `MODERATOR` role.
   # Mutation is restricted.
-  rejectUsername(input: RejectUsernameInput!): RejectUsernameResponse
+  unBanUser(input: UnBanUserInput!): UnBanUserResponse
```

Comment:
@wyatt can we change them to `unbanUser` and `unsuspendUser`. My rules are to captilize whenever there is a new word, but `unBan` and `unSuspend` does not follow this rule. It's more complicated to say _capitalize whenever there is a new word UNLESS the word marks the opposite effect (undo) of another action_ than just _capitalize whenever there is a new word_

Reply:
It's likely my bad english ha, I'll apply a refactor to resolve that.